### PR TITLE
Add Implementation

### PR DESCRIPTION
**Pull request by apfelbox**
Adds the first implementation.

### CODE REVIEW

File: js/lib/icons.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import {h} from "preact";
+
+
+/**
+ * For the dropdown indicator
+ */
+export function ChevronIcon ()
```
Comment (keichinger) [inline js/lib/icons.tsx]: Can we add return types?

File: scss/molecule/_current-tags.scss [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (keichinger) [inline scss/molecule/_current-tags.scss]: Can be removed.

**Comment by apfelbox** [comment]
Oh, apparently I forgot to clean up the `px` issue. Fixed now.

**Comment by apfelbox** [comment]
@keichinger all updated.

**Pull request merged**